cert_issuer/blockchain_handlers/ethereum/__init__.py

```diff
@@ -60,7 +60,7 @@ def instantiate_blockchain_handlers(app_config):
     if chain == Chain.mockchain:
         transaction_handler = MockTransactionHandler()
     # ethereum chains
-    elif chain == Chain.ethereum_mainnet or chain == Chain.ethereum_ropsten:
+    elif chain == Chain.ethereum_mainnet or chain == Chain.ethereum_ropsten or chain == Chain.ethereum_goerli or chain == Chain.ethereum_sepolia:
```

Comment:
could we change that to something like `chain in ethereum_chains` where `ethereum_chains` is an array with this list?